osu.Game/Overlays/BeatmapListingOverlay.cs

```diff
@@ -260,10 +314,29 @@ private void addContentToPlaceholder(Drawable content)
             currentContent.FadeIn(200, Easing.OutQuint);
         }
 
+        private void addPanelsToContent(IEnumerable<DirectPanel> panels)
+        {
+            // TODO: Fade in?
```

Comment:
As I wrote here, perhaps each panel should have a fade-in animation applied here?